src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -243,47 +247,50 @@ const InstanceSnapshots = (props: Props) => {
       />
       {hasSnapshots ? (
         <>
-          <Pagination
-            {...pagination}
-            id="pagination"
-            totalCount={instance.snapshots?.length ?? 0}
-            visibleCount={
-              filteredSnapshots.length === instance.snapshots?.length
-                ? pagination.pageData.length
-                : filteredSnapshots.length
-            }
-            selectedNotification={
-              selectedNames.length > 0 && (
-                <SelectedTableNotification
-                  totalCount={instance.snapshots?.length ?? 0}
-                  itemName="snapshot"
-                  parentName="instance"
-                  selectedNames={selectedNames}
-                  setSelectedNames={setSelectedNames}
-                  filteredNames={filteredSnapshots.map((item) => item.name)}
-                />
-              )
-            }
-            keyword="snapshot"
-          />
           <ScrollableTable
             dependencies={[filteredSnapshots, inTabNotification]}
+            tableId="instance-snapshot-table"
           >
-            <SelectableMainTable
-              headers={headers}
-              rows={pagination.pageData}
-              sortable
-              emptyStateMsg="No snapshot found matching this search"
+            <TablePagination
+              data={sortedRows}
+              id="pagination"
               itemName="snapshot"
-              parentName="instance"
-              selectedNames={selectedNames}
-              setSelectedNames={setSelectedNames}
-              processingNames={processingNames}
-              filteredNames={filteredSnapshots.map((snapshot) => snapshot.name)}
-              onUpdateSort={pagination.updateSort}
-              defaultSort="created_at"
-              defaultSortDirection="descending"
-            />
+              dataFiltered={
+                filteredSnapshots.length !== instance.snapshots?.length
+              }
+              displayTitle={
+                selectedNames.length > 0 && (
+                  <SelectedTableNotification
+                    totalCount={instance.snapshots?.length ?? 0}
+                    itemName="snapshot"
+                    parentName="instance"
+                    selectedNames={selectedNames}
+                    setSelectedNames={setSelectedNames}
+                    filteredNames={filteredSnapshots.map((item) => item.name)}
+                  />
+                )
+              }
+              pageLimits={[1, 2, 3]}
```

Comment:
I guess this is here for testing only.

Reply:
yes, this was the last component I tested, may have slipped through the craps when removing the prop